treeherder/model/models.py

```diff
@@ -204,7 +204,7 @@ def search(cls, search_term):
         time_limit = datetime.datetime.now() - datetime.timedelta(days=90)
 
         # Wrap search term so it is used as a phrase in the full-text search.
-        search_term_fulltext = '"%s"' % search_term.replace("\"", "")
+        search_term_fulltext = '"%s"' % search_term.replace("\"", "").replace("-", "")
```

Comment:
Should we switch the hyphens to spaces instead, so sheriffs that copy-paste an error line into a bug summary still get a match?